<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UGEngineControllWidget</class>
 <widget class="QMainWindow" name="UGEngineControllWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UGEngineControllWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>160</red>
         <green>160</green>
         <blue>160</blue>
        </color>
       </brush>
      </property>
      <property name="viewMode">
       <enum>QMdiArea::TabbedView</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="tabsMovable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreateConfig"/>
    <addaction name="actionLoadConfig"/>
    <addaction name="actionSaveConfig"/>
    <addaction name="actionCopyConfig"/>
    <addaction name="actionCloseConfig"/>
    <addaction name="separator"/>
    <addaction name="actionConfigOptions"/>
    <addaction name="actionApplicationOptions"/>
    <addaction name="actionClassesDescriptio"/>
    <addaction name="separator"/>
    <addaction name="actionCreateModel"/>
    <addaction name="actionLoadModel"/>
    <addaction name="actionSaveModel"/>
    <addaction name="separator"/>
    <addaction name="actionReloadParameters"/>
    <addaction name="separator"/>
    <addaction name="actionCaptureVideo"/>
    <addaction name="actionOpenVideoFile"/>
    <addaction name="actionOpenImage"/>
    <addaction name="separator"/>
    <addaction name="actionSaveCommonDescriptions"/>
    <addaction name="actionSaveClassesDescriptions"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuChannels">
    <property name="title">
     <string>Channels</string>
    </property>
    <addaction name="actionAddNew"/>
    <addaction name="actionInsert"/>
    <addaction name="actionDeleteLast"/>
    <addaction name="actionDeleteAll"/>
    <addaction name="actionClone"/>
   </widget>
   <widget class="QMenu" name="menuCalculate">
    <property name="title">
     <string>Calculate</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionPause"/>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionStep"/>
    <addaction name="actionRunNSteps"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="menuBroadcasters">
     <property name="title">
      <string>Broadcasters</string>
     </property>
     <addaction name="actionHttpBroadcasters"/>
     <addaction name="actionTcpBroadcaster"/>
     <addaction name="actionDevLineHttpBroadcaster"/>
    </widget>
    <addaction name="actionImagesFromWindow"/>
    <addaction name="separator"/>
    <addaction name="actionEngineMonitor"/>
    <addaction name="actionVideoSources"/>
    <addaction name="actionVideoRegistration"/>
    <addaction name="actionComponentsControl"/>
    <addaction name="actionChannelsControl"/>
    <addaction name="actionComponentsLinks"/>
    <addaction name="actionLogger"/>
    <addaction name="actionPerformance"/>
    <addaction name="actionTestCreator"/>
    <addaction name="actionFavoriteInformation"/>
    <addaction name="actionDrawEngine"/>
    <addaction name="actionWatchWindow"/>
    <addaction name="actionProfiling"/>
    <addaction name="actionVASimpleSettings"/>
    <addaction name="separator"/>
    <addaction name="menuBroadcasters"/>
    <addaction name="actionServerControl"/>
    <addaction name="actionTcpServer"/>
   </widget>
   <widget class="QMenu" name="menuNewWindow">
    <property name="title">
     <string>New Window</string>
    </property>
    <addaction name="actionImagesFromNewWindow"/>
    <addaction name="actionWatchesFromNewWindow"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuChannels"/>
   <addaction name="menuCalculate"/>
   <addaction name="menuWindow"/>
   <addaction name="menuNewWindow"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreateConfig"/>
   <addaction name="actionLoadConfig"/>
   <addaction name="actionSaveConfig"/>
   <addaction name="separator"/>
   <addaction name="actionReloadParameters"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionPause"/>
   <addaction name="actionReset"/>
   <addaction name="actionStep"/>
   <addaction name="separator"/>
   <addaction name="actionUpdateFromOtherthread"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDockWidget" name="dockWidgetComponentsList">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Components list</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidgetChannels">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>26</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>524287</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Chnls</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidgetLoger">
   <property name="windowTitle">
    <string>Logger</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetGraph">
   <property name="windowTitle">
    <string>Graph</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetProfiling">
   <property name="windowTitle">
    <string>Profiling</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <action name="actionLoadConfig">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Load Config</string>
   </property>
  </action>
  <action name="actionSaveConfig">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
  <action name="actionCopyConfig">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Copy Config</string>
   </property>
  </action>
  <action name="actionConfigOptions">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Config Options</string>
   </property>
  </action>
  <action name="actionApplicationOptions">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Application Options</string>
   </property>
  </action>
  <action name="actionClassesDescriptio">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Classes Descriptio</string>
   </property>
  </action>
  <action name="actionCreateModel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create Model</string>
   </property>
  </action>
  <action name="actionLoadModel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load Model</string>
   </property>
  </action>
  <action name="actionSaveModel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Model</string>
   </property>
  </action>
  <action name="actionReloadParameters">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reload Parameters</string>
   </property>
  </action>
  <action name="actionCaptureVideo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Capture Video</string>
   </property>
  </action>
  <action name="actionOpenVideoFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open Video File</string>
   </property>
  </action>
  <action name="actionOpenImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionSaveCommonDescriptions">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Common Descriptions</string>
   </property>
  </action>
  <action name="actionSaveClassesDescriptions">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Classes Descriptions</string>
   </property>
  </action>
  <action name="actionAddNew">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add New</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
  </action>
  <action name="actionDeleteLast">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete Last</string>
   </property>
  </action>
  <action name="actionDeleteAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete All</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </action>
  <action name="actionImagesFromWindow">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Images</string>
   </property>
  </action>
  <action name="actionEngineMonitor">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Engine Monitor</string>
   </property>
  </action>
  <action name="actionVideoSources">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Video Sources</string>
   </property>
  </action>
  <action name="actionVideoRegistration">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Video Registration</string>
   </property>
  </action>
  <action name="actionComponentsControl">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Components Control</string>
   </property>
  </action>
  <action name="actionComponentsLinks">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Components Links</string>
   </property>
  </action>
  <action name="actionPerformance">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Performance</string>
   </property>
  </action>
  <action name="actionFavoriteInformation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Favorite Information</string>
   </property>
  </action>
  <action name="actionDrawEngine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Draw Engine</string>
   </property>
  </action>
  <action name="actionWatchWindow">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Watch Window</string>
   </property>
  </action>
  <action name="actionVASimpleSettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Video Analytics settings</string>
   </property>
  </action>
  <action name="actionServerControl">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Server Control</string>
   </property>
  </action>
  <action name="actionHttpBroadcasters">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Http Broadcasters</string>
   </property>
  </action>
  <action name="actionTcpBroadcaster">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Tcp Broadcaster</string>
   </property>
  </action>
  <action name="actionCreateConfig">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Create Config</string>
   </property>
  </action>
  <action name="actionDevLineHttpBroadcaster">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>DevLine Http Broadcaster</string>
   </property>
  </action>
  <action name="actionImagesFromNewWindow">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Images</string>
   </property>
  </action>
  <action name="actionWatchesFromNewWindow">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Watches</string>
   </property>
  </action>
  <action name="actionUpdateFromOtherthread">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Update from other thread</string>
   </property>
  </action>
  <action name="actionChannelsControl">
   <property name="text">
    <string>Channels Control</string>
   </property>
   <property name="toolTip">
    <string>Channels Control</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Close Application</string>
   </property>
  </action>
  <action name="actionLogger">
   <property name="text">
    <string>Logger</string>
   </property>
  </action>
  <action name="actionTestCreator">
   <property name="text">
    <string>Test Creator</string>
   </property>
  </action>
  <action name="actionRunNSteps">
   <property name="text">
    <string>Run N steps</string>
   </property>
   <property name="toolTip">
    <string>Run N Steps</string>
   </property>
  </action>
  <action name="actionClone">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clone</string>
   </property>
  </action>
  <action name="actionProfiling">
   <property name="text">
    <string>Profiling</string>
   </property>
  </action>
  <action name="actionCloseConfig">
   <property name="text">
    <string>Close Config</string>
   </property>
  </action>
  <action name="actionTcpServer">
   <property name="text">
    <string>TcpServer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
